Library/Formula/haproxy.rb

```diff
@@ -16,13 +14,14 @@ class Haproxy < Formula
   depends_on "pcre"
 
   def install
-    args = ["TARGET=generic",
-            "USE_KQUEUE=1",
-            "USE_POLL=1",
-            "USE_PCRE=1",
-            "USE_OPENSSL=1",
-            "USE_ZLIB=1",
-            "ADDLIB=-lcrypto",
+    args = %w[
```

Comment:
I thought this should always be ` args = %W[ ` but the ` brew style ` command flagged it up as wrong?